ForumProject/notifications/utils.py

```diff
@@ -0,0 +1,25 @@
+from django.core.mail import send_mail
+from django.conf import settings
+import logging
+
+logger = logging.getLogger(__name__)
+
+def send_notification_email(subject, message, recipient_list):
+    """
+    Send a notification email to a list of recipients.
+    
+    Parameters:
+    - subject (str): The subject of the email.
+    - message (str): The body content of the email.
+    - recipient_list (list): List of email addresses to send to.
+    """
+    try:
+        send_mail(
+            subject,
+            message,
+            settings.EMAIL_HOST_USER,
+            recipient_list,
+            fail_silently=False
+        )
```

Comment:
Sending emails directly in a request-response cycle can significantly slow down response times. If recipient_list contains many emails, this will block the server process until all emails are sent

Reply:
Please see above